_posts/en/newsletters/2019-06-12-newsletter.md

```diff
@@ -38,10 +38,10 @@ changes to popular Bitcoin infrastructure projects.
       addresses, tracking when they've been paid, and finding or
       deriving the particular private keys necessary for spending.
 
-    - June 6th: discussion of the [consensus cleanup soft fork][xs
+    - <span id="cleanup-discussion" markdown="1">June 6th: discussion of the [consensus cleanup soft fork][xs
```

Comment:
How about:

```
    - 
      {:#cleanup-discussion}
      June 6th: discussion of the [consensus cleanup soft fork][xs
```

Same for `assume-utxo-demo` below.

Reply:
FWIW I couldn't get what John outlined to both work and pass the "things that should be on one line" check at the same.

Reply:
@bitschmidty yeah, I had the same problems when trying different variations.  However, FYI, Block Attribute Lists (BALs) can go either before or after the block you want them to act on, so I was able to put it after the first paragraph of that bullet:

```text
    - June 6th: discussion of the [consensus cleanup soft fork][xs
      cleanup sf] included its interaction with [bip-taproot][], whether
      parts of it should be dropped, and whether anything should be
      added.  ([More background][bg consensus cleanup])
      {:#cleanup-discussion}
```